package.json

```diff
@@ -152,6 +152,7 @@
     "markdown-it-emoji": "^2.0.2",
     "micromatch": "^4.0.5",
     "minimist": "^1.2.8",
+    "minisearch": "^6.0.1",
```

Comment:
IG this needs to be in dependencies as it is being referenced by client side code (the search will likely break if someone installs just vitepress, it's working on the deploy preview because there we install everything).